src/components/People.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import GhibliAPI from "./GhibliAPI";
+import "./People.css";
+
+export default class People extends Component {
+    constructor() {
+        super();
+        this.state = { find: "", people: [], person: { name: "default" } };
+    }
+    inputChange = (e) => {
+        this.setState({ find: e.target.value });
+    }
+    submitSearch = async (e) => {
+        e.preventDefault();
+        const { find, people } = this.state;
+        const lookingFor = find.trim().toLowerCase();
+        const found = people.filter(person => person.name.toLowerCase() === lookingFor);
```

Comment:
You basically invented JavaScript's `.find` method on arrays, here. Check it out if you don't know it, it would make this even sweeter than a `.filter` call!